content/cheatsheet.md

```diff
@@ -81,6 +81,20 @@ def hello_world():
   print "Hello World!"
 ```
 
+## Code
+
+Code can be displayed using the code shortcode as well.
```

Comment:
Is there a functional difference between this and code fences?

Reply:
I believe there is no difference. Personally, code fences are more readable so I suggest those are used (as they are now).

I documented the **code** shortcode so we have all the working shortcodes in one place and can remove it if you agree.